### PR TITLE
SC-960: Allow boolean values to be placed in yamls

### PR DESCRIPTION
**Pull request by silvanu**

### CODE REVIEW

File: Processor.php [hunk 1]
```diff
@@ -190,6 +190,14 @@ private function replaceValues(&$subject, $key, $replaceValuesMap) {
             case 'string':
                 // Replace values in subject
                 $subject = strtr($subject, $replaceValuesMap);
```
Comment (Eugst) [inline Processor.php]: move this line to default switch

Comment (silvanu) [inline Processor.php]: I need to do boolean check after translation.

Comment (Eugst) [inline Processor.php]:
```
if(array_key_exists($subject, $replaceValuesMap)) {
                    switch (strtolower($replaceValuesMap[$subject])) {
                        case 'true':
                            $subject = true;
                            break;
                        case 'false':
                            $subject = false;
                            break;
                        default:
                            $subject = strtr($subject, $replaceValuesMap);
                    }
                }
```

**Comment by Eugst** [comment]
:octocat:

**Comment by jaysphoto** [comment]
:octocat:

**Pull request merged**